src/oss-detect-backdoor/DetectBackdoorTool.cs

```diff
@@ -30,15 +32,34 @@ public class DetectBackdoorTool : OSSGadget
         /// </summary>
         private static NLog.ILogger Logger { get; set; } = NLog.LogManager.GetCurrentClassLogger();
 
-        /// <summary>
-        /// Command line options
-        /// </summary>
-        private readonly Dictionary<string, object?> Options = new Dictionary<string, object?>()
+        public class Options
         {
-            { "target", new List<string>() },
-            { "download-directory", null },
-            { "use-cache", false }
-        };
+#pragma warning disable CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable - commandlineparser doesnt handle nullable fields
```

Comment:
Why is nullable disabled here? This is a valid detection that DownloadDirectory should be a `string?` not a `string` the default value is even `null` in the command line options.